tests/unit/utils/test_time_formats.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Optional, Union
+
+from merlin.utils import convert_timestring
+
+
+def _list_all_equal(my_list: List) -> bool:
+    """Boolean check that all items in the list are the same"""
+    return all(item == my_list[0] for item in my_list)
+
+
+def check_all_equal(
```

Comment:
should this and some of the other utility functions also be protected like _list_all_equal() for style? I guess I don't follow what distinguishes the protected and public methods.